packages/analytics/analytics-chart/src/utils/colors.ts

```diff
@@ -81,16 +89,10 @@ export const apiRequestStatusCodeColors = {
 }
 
 export const statusCodeBadgeBackgroundColor = (statusCode: string) => {
-  // @ts-ignore TODO
-  return apiRequestStatusCodeColors[Math.floor(statusCode / 100) * 100]?.background || '#fafafa'
-}
-
-// TODO: Is this unused ?
+  const keyIndex = Math.floor(statusCode as any as number / 100) * 100
```

Comment:
@adorack I've typed the status code colors at the top of the file now. 

Even though I have the key as `statusCode: number`, I encounter trouble on the line above, which is why i've added `as any as number` :/

Reply:
You shouldn't need this cast -- why is the function attempting to divide `statusCode` by 100 if `statusCode` is actually a string?  Either the function takes a string and needs to `parseInt` it into a number, or it takes a number and doesn't need the cast...